pages/apim/3.x/kubernetes/apim-kubernetes-operator-quick-start.adoc

```diff
@@ -110,9 +122,7 @@ NOTE: For the Gateway to work with the GKO, ensure that the `services.sync.kuber
 To test the API, you can call it through the APIM Gateway by running the following command using your APIM Gateway URL:
 
 ....
-curl -i http://localhost:9000/gateway/k8s-basic-with-ctx
+curl -i http://<YOUR_GATEWAY_URL>/gateway/k8s-basic-with-ctx
```

Comment:
```suggestion
curl -i http://<gateway_host>:<gateway_port>/gateway/k8s-basic-with-ctx
```
And below say "Where `<gateway_host>` and `<gateway_port>` are the gateway host and port.